core/io/image_loader.h

```diff
@@ -37,10 +37,6 @@
 #include "core/os/file_access.h"
 #include "core/ustring.h"
 
-/**
-	@author Juan Linietsky <reduzio@gmail.com>
-*/
-
 /**
  * @class ImageScanLineLoader
  * @author Juan Linietsky <reduzio@gmail.com>
```

Comment:
I think you missed a spot.

Reply:
Needs rebase now that #31269 is merged.

Reply:
I have done the rebase and force pushed it.